brianlong/www.validators.app
# 186232848 stake account details

#### What's this PR do?
- create explorer stake account details page

#### How should this be manually tested?
- check all the links
- check history list
- check mobile view

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186232848)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
